options.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022 API7.ai, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloud
+
+import "time"
+
+var (
+	// DefaultOptions contains the default settings for all the options.
+	DefaultOptions = &Options{
+		ServerAddr:            "https://api.api7.cloud",
```

Comment:
Do we need to include `api/v1`?

Reply:
> Do we need to include api/v1?

It's just an address, without the path prefix, the path prefix `/api/v1` is the API version and the Go SDK will be consistent with it. We may have a V2 Go SDK and in such a case, will use `/api/v2`, so it's not suitable to put the path prefix to `ServerAddr`.

Reply:
> should this store be in env?

SDK should not consider this. It's the responsibility of caller of SDK .